### PR TITLE
Table class

### PR DESCRIPTION
**Pull request by AgnieszkaChS**

### CODE REVIEW

File: Table.cs [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        private void CreateRoundDeck()
+        {
+            roundDeck = new TableDeck();
+        }
+
+        private void CreateBenchDeck()
+        {
+            benchDeck = new TableDeck();
+        }
+
```
Comment (ntiany) [inline Table.cs]: Do we really need these two methods? We may initialize these decks in constructor or when creating fields

File: Table.cs [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        {
+            foreach (Player player in players)
+            {
+                if (player.localDeck.IsEmpty())
+                {
+                    return true;
+                }
+            }
+            return false;
+        }
```
Comment (ntiany) [inline Table.cs]: You may use LINQ method Any to shorten this method

File: Table.cs [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+            Player winner = players[0];
+            foreach (Player player in players)
+            {
+                if (player.GetNumberOfCardsInPlayersDeck() > winner.GetNumberOfCardsInPlayersDeck())
+                {
+                    winner = player;
+                }
+            }
+            return winner;
+        }
```
Comment (ntiany) [inline Table.cs]: You may use LINQ to shorten this method

File: Table.cs [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+        public TableDeck GetRoundDeck()
+        {
+            return roundDeck;
+        }
+
```
Comment (ntiany) [inline Table.cs]: Consider changing roundDeck to public property (private set) to get rid of this method

File: Table.cs (hunk 1 above)
Comment (AgnieszkaChS) [reply Table.cs]: Nope, already moved initialization to fields creation. Thanks!

**Review by chashafe — APPROVED** [review]

**Review by chashafe — APPROVED** [review]

**Review by ntiany — APPROVED** [review]

File: Table.cs (hunk 2 above)
Comment (AgnieszkaChS) [reply Table.cs]: Done!

File: Table.cs (hunk 3 above)
Comment (AgnieszkaChS) [reply Table.cs]: Done!

File: Table.cs (hunk 4 above)
Comment (AgnieszkaChS) [reply Table.cs]: Changed public property and moved roundDeck and benchDeck initialisation to Constructor

**Review by starczi — APPROVED** [review]

**Pull request merged**